people/handlers.go

```diff
@@ -20,15 +21,55 @@ func NewPeopleHandler(service PeopleService) PeopleHandler {
 }
 
 func (h *PeopleHandler) GetPeople(writer http.ResponseWriter, req *http.Request) {
+	writer.Header().Add("Content-Type", "application/json")
 	if !h.service.isInitialised() {
 		writeStatusServiceUnavailable(writer)
 		return
 	}
-	obj, found := h.service.getPeople()
-	writeJSONResponse(obj, found, writer)
+
+	if c, _ := h.service.getCount(); c == 0 {
+		writeJSONMessageWithStatus(writer, "People not found", http.StatusNotFound)
+		return
+	}
+
+	pv, err := h.service.getPeople()
+
+	if err != nil {
+		writeJSONMessageWithStatus(writer, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	defer pv.Close()
+	writer.WriteHeader(http.StatusOK)
+	log.Info("Status written")
+	io.Copy(writer, &pv)
+}
+
+func (h *PeopleHandler) GetPeopleByUUID(writer http.ResponseWriter, req *http.Request) {
```

Comment:
Calling this GetPeopleByUUID makes me think I'd need to pass in a UUID, which isn't the case. Perhaps GetPeopleUUIDs?